tests/integration/config/plugins/integration-tests/data/tests/tests_save_load.txt

```diff
@@ -1,3 +1,97 @@
+test-data "save load"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system "Terra Incognita"
+		planet Ruin
+		clearance
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 10
+				bunks 3
+				"cargo space" 50
+				drag 1
+				"engine capacity" 400
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 1300
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 200
+				"thrust" 50
+				"turn" 1000
+			outfits
+				Hyperdrive
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			# to skip the landing on ruin conversation
+			"Ruin: Landing: offered"
+
+
+mission "%TEST%: SAVE LOAD MID-CONVERSATION"
+	priority
+	source "Ruin"
+	to offer
+		has "%TEST%: SAVE LOAD MID-CONVERSATION"
+	on offer
+		conversation
+			`Test test`
+			action
+				"%TEST%: transient" = 1
+			choice
+				`	"Yes."`
+					accept
+
+test "Saving during conversation"
+	status active
+	description "Test that (automatically) saving during a conversation doesn't store state"
+	sequence
+		# Create/inject the savegame and load it.
+		inject "save load"
+		call "Load First Savegame"
+		apply
+			"%TEST%: SAVE LOAD MID-CONVERSATION" = 1
+		call "Depart"
+		assert
+			"%TEST%: transient" == 0
+		call "Land On Ruin helper"
+		call "Trigger Spaceport Mission"
+		assert
+			"%TEST%: transient" == 1
+		# Re-load savegame mid-conversation.
+		input
+			key escape
+		call "Load First Savegame"
+		assert
+			"%TEST%: transient" == 0
```

Comment:
Shouldnt we test that we can get that state back?
I assume that this means it was tested when we were landed before triggering the convo, right? So just try it again.

And then, reload *again* without saving to make sure it didnt auto save at the end of the convo? I think the code currently does that which is a bit weird if you ask me

Reply:
I mean eventually we could have an option where you choose when the game is saved

Reply:
I've added a section to the test going through the conversation again to see the mission was indeed triggered again after re-loading. Not entirely sure what you mean by the last bit about testing saving after the conversation though.

Reply:
Well simply trying to save (happens when you exit, right?) once the convo is done, load that save and see if the condition is still set

Reply:
Ah I see, that makes sense. Added now.